src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -547,15 +547,20 @@ private void populateSummaryStatisticsFromResponses(
             String recipientEmail = response.recipient;
 
             // Compute number of responses including user's self response
-            int numOfResponses = numResponses.getOrDefault(recipientEmail, 0) + 1;
-            numResponses.put(recipientEmail, numOfResponses);
+            numResponses.putIfAbsent(recipientEmail, 0);
+            numResponses.computeIfPresent(recipientEmail, (k, v) -> {
+                int numOfResponses = numResponses.getOrDefault(recipientEmail, 0) + 1;
+                return numOfResponses;
+            });
```

Comment:
Firstly the `recipientEmail` is not a parameter of the mapping function for `computeIfPresent` (I believe what are passed into that function is `k` and `v`).
Secondly the `computeIfPresent` and `getOrDefault` are redundant if the `putIfAbsent` in front ensures `recipientEmail` has a value. (and `computeIfPresent` is out of the scope of this issue).

lets not change this part. This part is quite succinct and readable.